Makefile

```diff
@@ -69,8 +69,9 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 	pip-compile --upgrade -o requirements/test.txt requirements/test.in
 	pip-compile --upgrade -o requirements/ci.txt requirements/ci.in
 	pip-compile --upgrade -o requirements/dev.txt requirements/dev.in
+	pip-compile --upgrade -o requirements/celery44.txt requirements/celery44.in
 	# Let tox control the Django version for tests
-	grep -e "^amqp==\|^anyjson==\|^billiard==\|^celery==\|^kombu==\|^click-didyoumean==\|^click-repl==\|^click==\|^prompt-toolkit==\|^vine==" requirements/base.txt > requirements/celery44.txt
+	grep -e "^amqp==\|^anyjson==\|^billiard==\|^celery==\|^kombu==\|^click-didyoumean==\|^click-repl==\|^click==\|^prompt-toolkit==\|^vine==" requirements/base.txt  > requirements/celery52.txt
```

Comment:
so, we don't want to maintain a `celery44` dependencies list?

Reply:
maybe we should've added a file for `celery50` instead of replacing `celery44` with `celery50`

Reply:
```suggestion
grep -e "^amqp==\|^anyjson==\|^billiard==\|^celery==\|^kombu==\|^click-didyoumean==\|^click-repl==\|^click==\|^prompt-toolkit==\|^vine==" requirements/base.txt > requirements/celery44.txt
grep -e "^amqp==\|^anyjson==\|^billiard==\|^celery==\|^kombu==\|^click-didyoumean==\|^click-repl==\|^click==\|^prompt-toolkit==\|^vine==" requirements/base.txt  > requirements/celery52.txt
```

Reply:
Latest requirements will only contain packages according to `celery52` so adding the suggested step will only create duplicate `celery44.txt` and `celery52.txt` files. 
In the make upgrade step, we can only fetch the requirements for the latest supported version so the requirements for `celery44.txt` are being generated with `celery44.in` now.